From d6e7804c8a4fd30672da902dd3f45ae59ec6f96b Mon Sep 17 00:00:00 2001
From: Matthew Fatheree <Matthew.Fatheree@belkin.com>
Date: Sun, 4 May 2014 20:43:34 +0700
Subject: [PATCH 23/34] mamba mvebu: board specific initialize script

 - Unexport GPIO33 (Reset button) in order can probe the gpio-key.ko
  driver for handle the behavour of Reset button by /dev/input/eventX.

Signed-off-by: Matthew Fatheree <Matthew.Fatheree@belkin.com>
---
 package/linksys-base-files/files/etc/init.d/mamba |    9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100755 package/linksys-base-files/files/etc/init.d/mamba

diff --git a/package/linksys-base-files/files/etc/init.d/mamba b/package/linksys-base-files/files/etc/init.d/mamba
new file mode 100755
index 0000000..7470a6e
--- /dev/null
+++ b/package/linksys-base-files/files/etc/init.d/mamba
@@ -0,0 +1,9 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006 OpenWrt.org
+
+START=09
+start() {
+	# Unexport GPIO33 of RESET button
+	echo 33 > /sys/class/gpio/unexport
+	sleep 1
+}
-- 
1.7.9.5

